### PR TITLE
Initial BrainPad VB document

### PR DESCRIPTION
**Pull request by greg-norris**

### CODE REVIEW

File: brainpad/courses/vb/intro.md [hunk 1]
```diff
@@ -1 +1,394 @@
-# Intro
+
+___
+# Introduction to the BrainPad 
+___
+
+___
+## Overview 
+Students will learn how to create projects in Visual Studio along with programming basics. Applications in this lesson will be limited to the **BrainPad** object, **If** statements, a **While** loop and the template functions.
```
Comment (Arke64) [inline brainpad/courses/vb/intro.md]: Put a period

File: brainpad/courses/vb/intro.md [hunk 2]
```diff
@@ -1 +1,394 @@
...
+## Install Visual Studio and Download .vsix file
+Install Visual Studio Community 2017. Select the Community download. Skip this step if you have Visual Studio 2017 installed.
+* [Visual Studio 2017 Download](https://www.visualstudio.com/downloads/)
+___
+
+## Starting a New Project
+Connect your BrainPad to your computer using a microUSB cable.
+
+Next Open Visual Studio select **File>New>Project** and under Other Languages select **Visual Basic>TinyCLR** and choose the **BrainPad Application**. Change the Location to  C:\Users\<**YourUsername**>\Desktop\ then name the project BrainPad_Project.
+
```
Comment (Arke64) [inline brainpad/courses/vb/intro.md]: Another period.

File: brainpad/courses/vb/intro.md (hunk 1 above)
Comment (greg-norris) [inline brainpad/courses/vb/intro.md]: Sweet

File: brainpad/courses/vb/intro.md (hunk 2 above)
Comment (greg-norris) [inline brainpad/courses/vb/intro.md]: Yes

**Pull request merged**